From 58b527d753a67d8dbda4b570c5b91ece6848330f Mon Sep 17 00:00:00 2001
From: madlep <madlep@ubercharged.net>
Date: Tue, 23 Dec 2008 23:44:44 +1100
Subject: [PATCH] fixed breaking activeresource unit tests

added local activesupport/lib to load path, and required activesupport.rb to explicitly load the version from the working copy rather than allowing later requies to fallback to whatever version is installed as a gem. I was getting a lot of breakages on Benchmark.ms, which is only in edge, and not 2.2.2
---
 activeresource/test/abstract_unit.rb |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/activeresource/test/abstract_unit.rb b/activeresource/test/abstract_unit.rb
index e612412..5fdad4a 100644
--- a/activeresource/test/abstract_unit.rb
+++ b/activeresource/test/abstract_unit.rb
@@ -1,5 +1,8 @@
 require 'test/unit'
 
+$:.unshift "#{File.dirname(__FILE__)}/../../activesupport/lib/"
+require 'activesupport'
+
 $:.unshift "#{File.dirname(__FILE__)}/../lib"
 require 'active_resource'
 require 'active_resource/http_mock'
-- 
1.6.0.4

